src/SFA.DAS.ProviderRegistrations.Web/Controllers/RegistrationController.cs

```diff
@@ -103,6 +102,7 @@ public async Task<IActionResult> InvitedEmployers(string sortColumn, string sort
             if (string.IsNullOrWhiteSpace(sortColumn)) sortColumn = Enum.GetNames(typeof(InvitationSortColumn)).First();
             if (string.IsNullOrWhiteSpace(sortDirection) || (sortDirection != "Asc" && sortDirection != "Desc")) sortDirection = "Asc";
 
+            //CB: Is this right? Should this not be fed in by IDAMS?

```

Comment:
Yes needs UkPrn but awaiting decision if this should be filtered by PRN or User.

Reply:
Updated Comment with a TODO